docs/topics/testing/tools.txt

```diff
@@ -1254,16 +1249,27 @@ The decorator can also be applied to test case classes::
 
     from django.test import TestCase, modify_settings
 
+    @modify_settings(MIDDLEWARE={
```

Comment:
So it's this example that I think is out of context, and will lead to confusions. 

Ostensibly this example shows how `@modify_settings` can be applied at the class level. The nesting is a _What?_ here. 

(Better, I'm trying to say, would be to have another example inside the `note::` block demonstrating it there, if an example is needed. That way it would be clear at least.)

Reply:
Oh right - TBH, I didn't even see that, it was already applied on class-level.. it's a bit against my own intuition to have decorators be so... dynamically typed.

But I get the point here that it's overloading the example with a bit too much complexity, bearing in mind that the reader should be able to grasp the many ways that the decorator can be applied.

Reply:
Shorted and force-pushed another commit! Thanks for the input and for repeating it so I could understand precisely what you meant! :)